packages/textlint-fixer-formatter/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "textlint-fixer-formatter",
+  "version": "1.0.0",
+  "description": "TBD",
```

Comment:
> textlint output formatter for fixer

Reply:
@azu Thank you for your suggestion.
I'll update description like your comment.